lib/commands/general.js

```diff
@@ -49,6 +49,16 @@ commands.getWindowSize = async function () {
   return await this.uiautomator2.jwproxy.command('/window/current/size', 'GET', {});
 };
 
+// For W3C
+commands.getWindowRect = async function () {
+  const rect = await this.getWindowSize();
```

Comment:
This seems odd. The size is returned by `this.getWindowSize()`, not the rect. So it should be more like
```js
const {width, height} = await this.getWindowSize();
return {
  width,
  height,
  x: 0,
  y: 0,
};
```